gson/src/test/java/com/google/gson/internal/LazilyParsedNumberTestCase.java

```diff
@@ -0,0 +1,93 @@
+package com.google.gson.internal;
+
+import com.diffblue.deeptestutils.Reflector;
+import com.google.gson.internal.LazilyParsedNumber;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import java.lang.reflect.InvocationTargetException;
+
+public class LazilyParsedNumberTestCase {
+
+  @Rule public ExpectedException thrown = ExpectedException.none();
+
+  /* testedClasses: com/google/gson/internal/LazilyParsedNumber.java */
+  /*
+   * Test generated by Diffblue Deeptest.
+   * This test case covers:
+   *  - conditional line 87 branch to line 90
+   *  - conditional line 90 branch to line 91
+   *  - conditional line 91 branch to line 91
+   *  - conditional line 92 branch to line 92
+   */
+
+  @Test
+  public void equalsInputNotNullOutputFalse() throws InvocationTargetException {
+
+    // Arrange
+    final LazilyParsedNumber objectUnderTest =
+        ((LazilyParsedNumber)Reflector.getInstance("com.google.gson.internal.LazilyParsedNumber"));
+    Reflector.setField(objectUnderTest, "value", "\"");
```

Comment:
Not sure what LazilyParsedNumber is exactly doing, but 'Number' would indicate that 'value' should be a number, preferably. Please add this to the heuristics ticket.

Reply:
OK